src/document-parser.ts

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Yea, I guess parser is already a used term in our tooling/build ecosystem.
Changing to `get-dependencies-from-document.ts`